stubs/psycopg2/psycopg2/_psycopg.pyi

```diff
@@ -109,7 +109,7 @@ class cursor:
     def __exit__(
         self, type: type[BaseException] | None, value: BaseException | None, traceback: TracebackType | None
     ) -> None: ...
-    def __iter__(self: Self) -> Self: ...
+    def __iter__(self) -> Iterator[tuple[Any, ...]]: ...
```

Comment:
Just for reference from the official docs: https://www.psycopg.org/docs/cursor.html#fetch